src/shared/modals/custom-filter/FilterTabs.js

```diff
@@ -27,6 +27,34 @@ const FilterTabs = (props) => {
     setDisplayedRows
   } = props
 
+  const renderFilterErr = () => {
+    return (
+      <div className="filter__search-error">
+        <div>No Results</div>
+        <p>Sorry we didn’t find any row titles that match your search.</p>
```

Comment:
I would update this text to say, "Sorry, we didn't find any [rows/columns (depending on the tab)] that match your search."